content/docs/guide-development.md

```diff
@@ -24,13 +24,26 @@ You can use a mock [lightcurve notebook](https://renkulab.io/gitlab/astronomy/mm
 ### Parametetrize the notebook 
 
 * create a cell with the following tag "parameters" (see [papermill manual](https://papermill.readthedocs.io/en/latest/usage-parameterize.html#designate-parameters-for-a-cell)):
+  * if not annotated, the types of the inputs parameters are determined based on the parameter default value
+  * one can annotate the input parameter by putting comment with the `term` from the [ontology](https://odahub.io/docs/guide-ontology).
+  * several parameters are mandatory in the MMODA platform. These includes:
+    | Type annotation | Parameter default name |
+    | ---------------- | -------------- |
+    | http://odahub.io/ontology#PointOfInterestRA | RA |
+    | http://odahub.io/ontology#PointOfInterestDEC | DEC |
+    | http://odahub.io/ontology#StartTime | T1 |
+    | http://odahub.io/ontology#EndTime | T2 |
+    | http://odahub.io/ontology#AstrophysicalObject | src_name |
```

Comment:
it does not make too much sense to mandate src_name and coordinates. but it is so because resolving is done by the frontend.

Reply:
Actually, I never used the src_name in any of the plugins I've made

Reply:
> Maybe we could fix the description for the current state of the art for Andrii to be able to experiment. Then we may discuss on Monday and make another iteration.

The discussion is not only in updates on Monday, these comments here are part of the feedback for further interations.
If it seems too complex to do it right straight away (I did not realize it would be), let's make issues like I did in some cases.

Reply:
> Actually, I never used the src_name in any of the plugins I've made

ok, so it should be clear for future developers how to deal with this

https://github.com/oda-hub/hugo-odahub/issues/12